tests/extmod/select_poll_fd.py

```diff
@@ -34,11 +34,22 @@
 # Poll for input, should return an empty list.
 print(poller.poll(0))
 
-# Test registering a very large number of file descriptors.
+# Test registering a very large number of file descriptors (will trigger
+# EINVAL due to more than OPEN_MAX fds).
 poller = select.poll()
-for fd in range(6000):
+for fd in range(1, 6000):
     poller.register(fd)
 try:
     poller.poll()
+    assert False
```

Comment:
This assertion is now failing in the tests, when run on CPython.  Not sure why, maybe the introduction of the `1,` in the range above, or maybe this never worked correctly on CI??  The aim of this test was for coverage of error cases, and I'm pretty sure it was working correctly on CI (ie hitting the OSError) because the coverage was good.

Maybe increase the 6000 to 10000?

Reply:
I changed it to just `range(10000)`, see what the CI says.

Reply:
CI is still not happy.